sw/device/benchmarks/coremark/README.md

```diff
@@ -1,13 +1,28 @@
-# Options
-There are two main options available for coremark SIM, ITERATIONS
+# Building Coremark
 
-## ITERATIONS
-Controls how many iteartions are run.  By default this value is 0 and will cause coremark to run >10s.
-For simulation and verilator, set to 1 for a reduced run
+To build coremark under meson:
 
-## For verilator
-make distclean; make SIM=1 ITERATIONS=1
-SIM=1 matces the verilator UART baudrates
+```sh
+cd "${REPO_TOP}"
+./meson_init.sh -f
+ninja -C build-out/sw/${TARGET} sw/device/benchmarks/coremark/coremark.bin
+```
 
-## For DV / FPGA
-make distclean; make ITERATIONS=1/N
+Where ${TARGET} is one of 'sim-verilator' or 'fpga'
+
+This will give you a .bin and .vmem file (suitable for either spiflash or
+giving directly to --flashinit for verilator) which can be found in
+`build-out/sw/${TARGET}/sw/device/benchmarks/coremark`
```

Comment:
Please don't rely on any binary path in build-out. Use the ones in build-bin instead.